src/taipy/core/data/excel.py

```diff
@@ -287,20 +287,29 @@ def _read_as_modin_dataframe(
         except pd.errors.EmptyDataError:
             return modin_pd.DataFrame()
 
+    def __write_excel_with_sheet_name(self, write_excel_fct, *args, **kwargs):
```

Comment:
I would rename the method `__do_write` since we don't necessarily write with a sheet name

Reply:
I have renamed it to `__write_excel_with_single_sheet` :D

Reply:
I love it